04-networking.md

```diff
@@ -63,7 +63,7 @@ The following two resource groups will be created and populated with networking
 
    ```bash
    # [This takes about five minutes to run.]
-   az deployment group create -g rg-enterprise-networking-hubs-$TEAM_NAME -f networking/hub-default.json -p location=eastus2
+   az deployment group create -g rg-enterprise-networking-hubs-$TEAM_NAME -f networking/hub-default.json -p location=$TEAM_LOCATION
```

Comment:
Here we're creating the hub but we told them that we're using a central hub.  Let's stay with per team hubs, but fix the description that says it's one hub.